docs/site/Components.md

```diff
@@ -0,0 +1,52 @@
+---
+lang: en
+title: 'Components'
+keywords: LoopBack 4.0, LoopBack 4
+sidebar: lb4_sidebar
+permalink: /doc/en/lb4/Components.html
+---
+
+## Overview
+
+Components play an important part in the extensibility of LoopBack 4. A
+Component makes it easy for independent developers to contribute additional
+features to LoopBack. Components serve as a vehicle to group extension
+contributions such as [Context Bindings](Context.md) and various artifacts to
+allow easier extensibility of your Application.
+
+A typical LoopBack component is an [npm](https://www.npmjs.com) package
+exporting a Component class which can be added to your application.
+
+Apart from its own properties, `Component` class can have the following
+properties:
```

Comment:
Since this page is in `Key Concepts`, I think we should stay at high level and just list the kinds of artifacts that a component can contribute. No need to go into implementation details and component class properties, such content belongs to [Extending LoopBack 4](https://loopback.io/doc/en/lb4/Extending-LoopBack-4.html) and more specifically to [Creating components](https://loopback.io/doc/en/lb4/Creating-components.html).

I guess we can leave this improvement out of scope of this pull request if you prefer.